### PR TITLE
fixed glitches in landing page

### PR DESCRIPTION
**Pull request by justiceotuya**

### CODE REVIEW

File: components/Layout/components/PageLayout.css [hunk 1]
```diff
@@ -3,6 +3,10 @@
     margin-top: 64px;
 }
 
+.PageLayout_content_sidebar{
+height: calc(100vh - 64px);
```
Comment (enyeInc) [inline components/Layout/components/PageLayout.css]: fix indent on line 7 and space before `{` on line 6

**Review by enyeInc — DISMISSED** [review]
LGTM

File: components/Authentication/components/Login.jsx [hunk 2]
```diff
@@ -73,7 +74,7 @@ class NormalLoginForm extends React.Component {
                         {/* buttons */}
                         <Form.Item>
                             <a className="login-form-forgot" href="/password_reset">Forgot password</a>
-                            <Button type="primary" htmlType="submit" className="login-form-button" loading={loading} onClick={this.enterLoading}>
```
Comment (EngrEric) [inline components/Authentication/components/Login.jsx:76]: Its a better practice to move all strings to a constants

**Review by EngrEric — CHANGES_REQUESTED** [review]
Looks good, try to move all strings to constant because it does not only make your code look good but also it increases performance infinitesimally

File: components/Authentication/components/Login.jsx (hunk 2 above)
Comment (justiceotuya) [reply components/Authentication/components/Login.jsx:76]: Fixed

**Review by enyeInc — APPROVED** [review]

**Pull request merged**